templates/CMakeLists.txt.template

```diff
@@ -260,8 +260,6 @@
         WORKING_DIRECTORY <%text>${CMAKE_CURRENT_SOURCE_DIR}</%text>
         COMMENT "Running gRPC C++ protocol buffer compiler on <%text>${FIL}</%text>"
         VERBATIM)
-
-        <%text>set_source_files_properties("${_gRPC_PROTO_GENS_DIR}/${RELFIL_WE}.grpc.pb.cc" "${_gRPC_PROTO_GENS_DIR}/${RELFIL_WE}.grpc.pb.h"  "${_gRPC_PROTO_GENS_DIR}/${RELFIL_WE}_mock.grpc.pb.h" "${_gRPC_PROTO_GENS_DIR}/${RELFIL_WE}.pb.cc" "${_gRPC_PROTO_GENS_DIR}/${RELFIL_WE}.pb.h" PROPERTIES GENERATED TRUE)</%text>
```

Comment:
why remove this line?

Reply:
It's not needed. Any file generated by [`add_custom_command()`](https://cmake.org/cmake/help/latest/command/add_custom_command.html) is automatically marked as [`GENERATED`](https://cmake.org/cmake/help/latest/prop_sf/GENERATED.html).